pass_import.py

```diff
@@ -184,11 +186,32 @@ def insert(self, path, data, force=False):
         """Multiline insertion into the password store."""
         if not force:
             if os.path.isfile(os.path.join(self.prefix, path + '.gpg')):
-                raise PasswordStoreError("An entry already exists for %s." % path)
+                raise PasswordStoreError(
```

Comment:
Not in the scope of this PR. (Same for all similar changes)

BTW, the max-line-length in this project is 99 anyway. See `.prospector.yml`